### PR TITLE
{Doc} Update edge doc

### PR DESCRIPTION
**Pull request by bebound**
**Description**

Users need to uninstall official version before installing edge build.
If the official and edge versions are not the same, installing the newer version uninstalls the old version automatically.
If the versions are the same, installing edge does not remove unnecessary files and leads to error, like #25607

This PR also fix the dead links.

**Testing Guide**
You can keep two builds with same version on your machine.
image

**History Notes**

[Component Name 1] BREAKING CHANGE: `az command a`: Make some customer-facing breaking change
[Component Name 2] `az command b`: Add some customer-facing feature

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [ ] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -167,13 +169,13 @@ brew install $(curl -Ls -o /dev/null -w %{url_effective} https://aka.ms/InstallA
...
-curl -Ls -o azure-cli_xenial_all.deb https://aka.ms/InstallAzureCliXenialEdge && dpkg -i azure-cli_xenial_all.deb
+curl --location --silent --output azure-cli_jammy.deb https://aka.ms/InstallAzureCliJammyEdge && dpkg -i azure-cli_jammy.deb
 ```
 
-And install the edge build with rpm package on CentOS/RHEL/Fedora:
+And install the edge build with rpm package on RHEL 8 or CentOS Stream 8:
 
 ```bash
-rpm -ivh --nodeps $(curl -Ls -o /dev/null -w %{url_effective} https://aka.ms/InstallAzureCliRpmEdge)
+dnf install -y $(curl --location --silent --output /dev/null --write-out %{url_effective} https://aka.ms/InstallAzureCliRpmEl8Edge)
```
Comment (bebound) [inline README.md:178]: `--nodeps` does not install dependency `python39`, use `dnf` instead.

**Comment by yonzhan** [comment]
Update edge doc

File: README.md [hunk 2]
```diff
@@ -158,6 +158,8 @@ You can download the latest builds by following the links below:
 | Ubuntu Jammy Deb  | https://aka.ms/InstallAzureCliJammyEdge    |
 |      RPM el8      | https://aka.ms/InstallAzureCliRpmEl8Edge   |
 
+On Windows, you need to uninstall the official version before installing the edge build. (see: https://github.com/Azure/azure-cli/issues/25607#issuecomment-1452855212)
```
Comment (jiasli) [inline README.md:161]:
```suggestion
On Windows, you need to uninstall the official version before installing the edge build. (See https://github.com/Azure/azure-cli/issues/25607#issuecomment-1452855212)
```

File: README.md [hunk 3]
```diff
@@ -167,13 +169,13 @@ brew install $(curl -Ls -o /dev/null -w %{url_effective} https://aka.ms/InstallA
 You can install the edge build on Ubuntu Xenial with the following command:
 
 ```bash
-curl -Ls -o azure-cli_xenial_all.deb https://aka.ms/InstallAzureCliXenialEdge && dpkg -i azure-cli_xenial_all.deb
+curl --location --silent --output azure-cli_jammy.deb https://aka.ms/InstallAzureCliJammyEdge && dpkg -i azure-cli_jammy.deb
```
Comment (jiasli) [inline README.md:172]: L169 should also be updated.

Comment (jiasli) [inline README.md:172]: Should we distinguish between AMD64 and ARM64?

Comment (bebound) [reply README.md:172]: x86 architecture is still the mainstream, and AMD64 version should be enough for edge test.

**Review by jiasli — APPROVED** [review]

**Pull request merged**